components/commander/common/commander_frontend_delegate.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_COMMANDER_COMMON_COMMANDER_FRONTEND_DELEGATE_H_
+#define BRAVE_COMPONENTS_COMMANDER_COMMON_COMMANDER_FRONTEND_DELEGATE_H_
+
+#include <string>
+#include <vector>
+
+#include "base/component_export.h"
+#include "base/observer_list_types.h"
+#include "brave/components/commander/common/commander_item_model.h"
+
+namespace commander {
+class COMPONENT_EXPORT(COMMANDER_COMMON) CommanderFrontendDelegate {
```

Comment:
I wonder why `CommanderFrontendDelegate` is located in `common` component, instead of `browser` component.

Reply:
Moved to `commander/browser` component.